sktime/annotation/tests/test_all_annotators.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import pandas as pd
+import numpy as np
+from sktime.annotation.base._base import BaseAnnotator
+
+
+class TestAnnotator(BaseAnnotator):
```

Comment:
create mock annotator class in `sktime/annotation/base/_mock.py` to make sure all the unit testing functionality works
```python
class MockAnnotator(BaseAnnotator):
    # mock class, implement basic interface fit/predict
```